src/index.css

```diff
@@ -1444,16 +1444,16 @@ input:-webkit-autofill {
 }
 
 .footer-links__item~*:before {
-  padding: 0 0.6em 0 0.5em;
+  padding: 0 0.8em 0 0.8em;
   content: "|";
   font-size: 1em;
 }
 
 .footer-logo {
-  float: right;
+  float: left;
   height: 32px;
-  width: 127px;
-  margin-top: 22px;
+  margin-top: 15px;
+  margin-right: 35px;
 }
```

Comment:
When possible, I prefer to use inline style objects instead of relying on css class names. Here are some docs that describe this approach: https://reactjs.org/docs/dom-elements.html#style

For this PR, don't worry, it's not a requirement. For future work, we'll try to move towards using style objects instead of css classes. Some of my favorite reasons for this:

1. Style objects can be refactored easily with modern IDEs, but css is more difficult.
2. Style objects can be easily merged/manipulated with event handlers.
3. Style objects can live close to the rendered objects and are less likely to become orphaned. 
4. CSS classes, on the other hand, are a maintenance headache.